Ark.Tools.FtpClient.ArxOne/FtpClientPoolArxOneWithSocksAndConfigurer.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) 2023 Ark Energy S.r.l. All rights reserved.
+// Licensed under the MIT License. See LICENSE file for license information. 
+using Ark.Tools.FtpClient.Core;
+using ArxOne.Ftp;
+using Org.Mentalis.Network.ProxySocket;
+using System;
+using System.Net;
+using System.Net.Sockets;
+
+namespace Ark.Tools.FtpClient
+{
+    public class FtpClientPoolArxOneWithSocksAndConfigurer : FtpClientPoolArxOne
```

Comment:
just extend the FtpClientPoolArkOne with multiple ctor. that is not supposed to be registered in containers with 'default' ctors.